### PR TITLE
Storefront eslint

### PR DESCRIPTION
**Pull request by AydinHassan**
bit like same problem we had in platform, maybe we wanna do this different?

### CODE REVIEW

File: .github/workflows/storefront-eslint.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        run: npm install --no-audit --no-fund --prefer-offline
+
+      - name: Install Plugin Dependencies
+        working-directory: custom/plugins/${{ inputs.extensionName }}/src/Resources/app/storefront
+        run: npm install --no-audit --no-fund --prefer-offline
+
+      - name: Run ESLint
+        working-directory: custom/plugins/${{ inputs.extensionName }}/src/Resources/app/storefront
+        run: |
+          ../../../../../../../src/Storefront/Resources/app/storefront/node_modules/.bin/eslint --config ../../../../../../../src/Storefront/Resources/app/storefront/.eslintrc.js ./src
```
Comment (shyim) [inline .github/workflows/storefront-eslint.yml:42]:
```suggestion
          ${{ github.workspace }}/src/Storefront/Resources/app/storefront/node_modules/.bin/eslint --config ${{ github.workspace }}src/Storefront/Resources/app/storefront/.eslintrc.js ./src
```

Comment (shyim) [reply .github/workflows/storefront-eslint.yml:42]: maybe plugins should bring their own config 😅

Comment (shyim) [reply .github/workflows/storefront-eslint.yml:42]: Maybe we could also switch to https://biomejs.dev 🤔

Comment (AydinHassan) [reply .github/workflows/storefront-eslint.yml]: nice - maybe we can suggest it

**Comment by pweyck** [comment]
Maybe we can generalize it a bit and add it to https://github.com/shopware/github-actions/blob/main/eslint/action.yml

**Comment by pweyck** [comment]
https://github.com/shopware/github-actions/pull/23 change should cover it (the lint npm script might require a change)

**Pull request closed without merging**